src/routes/api/api-resync-post.ts

```diff
@@ -4,6 +4,22 @@ import { GitHubServerApp } from "models/github-server-app";
 import { Subscription } from "models/subscription";
 import { findOrStartSync } from "~/src/sync/sync-utils";
 import { serializeSubscription } from "routes/api/api-utils";
+import { Installation } from "models/installation";
+
+const getIdsForExistingInstallations = async(installationIds, logger): Promise<number[]> => {
+	const installations = installationIds.map(async(id: number): Promise<number | null> => {
+		const installation: Installation | null = await Installation.getForId(id);
```

Comment:
Also, if find by the `id` (pk), I think we can do `Installation.findByPk` from the buildin method.